subworkflows/local/amp.nf

```diff
@@ -17,6 +20,7 @@ workflow AMP {
     fastas // tuple val(meta), path(contigs)
     faas   // tuple val(meta), path(PROKKA/PRODIGAL.out.faa)
     tsvs   // tuple val(meta), path(MMSEQS_CREATETSV.out.tsv)
+    gbks   // tuple val(meta), path(ANNOTATION/ANNOTATION_TOOL.out.gbk)
```

Comment:
```suggestion
    gbks   // tuple val(meta), path(ANNOTATION_ANNOTATION_TOOL.out.gbk)
```
Could also write `PROKKA/PRODIGAL.out.gbk` like above, since this comment is only for dev. I don't mind :shrug: